.github/workflows/remove-automerge-label.yml

```
@@ -0,0 +1,33 @@
+name: Remove Auto-Merge Label on New Commit
+
+on:
+  pull_request:
```

```suggestion
  pull_request_target:
```

for write permission

---

TIL, good catch